cortex/kernel_features/kv_cache_manager.py

```
@@ -0,0 +1,157 @@
+
+
+class SharedMemoryPool:
+    def __init__(self, name: str, size: int, create: bool = True):
+        self.name = f"{SHM_PREFIX}{name}"
+        self.size = size
+        if create:
+            try:
+                old = shared_memory.SharedMemory(name=self.name)
+                old.close()
+                old.unlink()
+            except:
```

Ruff (0.14.7)

Do not use bare `except`

(E722)